src/lib/updating-element.ts

```diff
@@ -211,10 +211,11 @@ export abstract class UpdatingElement extends HTMLElement {
   private static _attributeToPropertyMap: AttributeMap;
 
   /**
-   * Marks class as having finished creating properties.
+   * This property was previously used to record the classes that have already
+   * been finalized. This property is no longer used, but is still declared here
+   * for backwards compatability in TypeScript typings.
```

Comment:
IMO breaking TypeScript typings isn't a breaking change for the library.

It would be better for someone to know that the finalized property no longer means anything. In any event, I doubt this will affect anyone, as the finalized property is pretty inside baseball.

Reply:
Fair enough, removed.